### ISSUE TITLE
 make robot acct token retrievable at any time

### ISSUE THREAD

**Issue by xaleeks** [goharbor/harbor#10096]
<img width="787" alt="Screen Shot 2019-11-24 at 6 05 15 PM" src="https://user-images.githubusercontent.com/51319901/70021035-309da100-155d-11ea-9d9d-cf2494aee7f7.png">

especially given there is no way to regen token right now, we need a way for user to be able to access this after creation

**Comment by reasonerjt** [comment]
Since this is a token it can be retrived only once.

**Comment by esantoro** [comment]
It would be nice to have a way to authenticate a token though, without resorting to interacting with the underlying registry.

We have some additional glue code to automate part of registry management and interaction from CI, and it would be nice to have a way to authenticate a robot account against a project.
